RebuyFormsRating/Models/RatingViewHandler.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using RebuyFormsRating.Common.Services;
+using Xamarin.Forms;
+using Acr.UserDialogs;
+using System.Threading.Tasks;
+
+namespace RebuyFormsRating.Models
+{
+    public class RatingViewHandler
+    {
+        public bool IsRated {
+            get { return Settings.IsRated; }
+            set { Settings.IsRated = value; }
+        }
+
+        public int UsesBeforeRating { set; get; } = 1;
+
+        public int UsageCount {
+            get { return Settings.UsageCount; }
+            set { Settings.UsageCount = value; }
+        }
+
+        public string VersionNumber {
+            get { return Settings.VersionNumber; }
+            set { Settings.VersionNumber = value; }
+        }
+
+        public void CheckOpenRatingView(string appStoreId)
+        {
+            if (String.IsNullOrWhiteSpace(VersionNumber) || !VersionNumber.Equals(DependencyService.Get<IInfoService>().AppVersionCode)) {
+                resetReminder();
+            } else {
+                if (!IsRated && UsageCount >= UsesBeforeRating) {
+                    showActionSheet(appStoreId);
+                }
+
+                UsageCount++;
+            }
+        }
+
+        private async Task showActionSheet(string appStoreId)
+        {
+            var action = await UserDialogs.Instance.ActionSheetAsync(
```

Comment:
It would be really nice, but perhaps not in-scope, to use the localization functionality of Xamarin.Forms to provide both English and German implementations of the strings.  As an outsider, I'd assume the implementation was in English, since the README is in English.  If not possible, then perhaps a note in the README as to what the strings need to be updated for an English implementation. 

https://developer.xamarin.com/guides/xamarin-forms/localization/